website/docs/sdk-reference/php.md

```diff
@@ -177,25 +303,61 @@ You can use the following caching options:
 ## Logging
 The SDK uses the PSR-3 `LoggerInterface` for logging, so you can use any implementor package like [Monolog](https://github.com/Seldaek/monolog).
 ```php
-
 $client = new \ConfigCat\ConfigCatClient("#YOUR-SDK-KEY#", [
-    'logger' => new \Monolog\Logger("name"),
+    \ConfigCat\ClientOptions::LOGGER => new \Monolog\Logger("name"),
 ]);
 ```
-
-## HTTP Client (Proxy)
-The SDK uses [Guzzle](http://docs.guzzlephp.org/en/stable/index.html) for the underlying HTTP calls and its request options are available to customize through the SDK options like:
+You can change the verbosity of the logs by setting the `log-level` configuration option.
 ```php
+$client = new \ConfigCat\ConfigCatClient("#YOUR-SDK-KEY#", [
+    \ConfigCat\ClientOptions::LOG_LEVEL => \ConfigCat\Log\LogLevel::INFO
+]);
+```
+
+The following levels are used by the ConfigCat SDK:
+
+| Level      | Description                                                                             |
+| ---------- | --------------------------------------------------------------------------------------- |
+| `NO_LOG`   | Turn the logging off.                                                         |
+| `ERROR`    | Only error level events are logged.                                                     |
+| `WARNING`  | Errors and Warnings are logged.                                                         |
+| `INFO`     | Errors, Warnings and feature flag evaluation is logged.                                 |
+| `DEBUG`    | All of the above plus debug info is logged. Debug logs can be different for other SDKs. |
 
+> The default level is `WARNING`. With `NO_LOG` you can turn the logging completely off.
+
+Info level logging helps to inspect the feature flag evaluation process:
+```bash
+[2022-01-06T18:34:16.846039+00:00] ConfigCat.INFO: Evaluating getValue(isPOCFeatureEnabled).
+User object: {"Identifier":"435170f4-8a8b-4b67-a723-505ac7cdea92","Email":"john@example.com"}
+Evaluating rule: [Email:john@example.com] [CONTAINS] [@something.com] => no match.
+Evaluating rule: [Email:john@example.com] [CONTAINS] [@example.com] => match, returning: true.
+```
+
+## HTTP Client
+The SDK uses [Guzzle](http://docs.guzzlephp.org/en/stable/index.html) for the underlying HTTP calls and its request options are available to customize through SDK options.
```

Comment:
I suggest explaining:
Why would I want to do it?
How to do it?

Reply:
The next 2 sections below explain what you can do in which scenarios